### ISSUE TITLE
SteamVR's new IVRInput system with 2.8.0_dev version

### ISSUE THREAD

**Issue by indeu** [OpenVR-Advanced-Settings/OpenVR-AdvancedSettings#103]
Dear developers,

I was very happy when I found out about your 2.8.0 release version.
I don't want to use matzcat666's 2.5.0 version.

Anyway now I am testing your developing version. 
Because I'm interested in the rotation for seated position. (kung-vr's branch works)
I built the source code one week's ago with QT and VS2015

After I set the drag or shortcutLeft/Right, It doesn't work. 
Is there any works for combine with new SteamVR's input system? 
(Sorry, I'm not understood about steamVR input system yet. It works fine about 2.8.0 drag moves X-Y-Z)

**Comment by ykeara** [comment]
We are actively gearing up for 3.0.0 release here in the next 7-14 days probably (a few more minor changes, and testing to do)[no guarantees ofc]

you can go to http://127.0.0.1:8998/dashboard/controllerbinding.html while steamVR is running, to access the UI via desktop (just makes things easier).

Just to make Turn = rotation, Drag = moving.

You will need to set them to a Button On your controller, (we will release a steamVR input guide along with 3.0.0).

IN ADDITION, in the overlay you need to make sure you have the checkbox's for enabling room turn and/or drag  are checked for what you want to use.... (and sorry the names are in flux so they are probably different on your version than what is most current.)

If that doesn't help you Either I need more information or @Kung-vr will have to respond.

**Comment by indeu** [comment]
Thanks. 

I already access http://127.0.0.1:8998/dashboard/controllerbinding.html page, 
but I didn't know how to setup that. I could not find drag and turn actions. 

Finally, I found the way. 
There are so many apps in the controller binding page. (I always selected steamVR or compositor <- wrong way)

This is how to bind key for turn or bind.
 - launch the controller binding setup page 
 - select the 'ADVANCED_SETUP.EXE' for combine with App
 - click the edit (left top side)
 - select the button for using turn or drag.
 - It may saved automatically. 

Yesterday, I played the ArizonaSunshine well on seated position. It works perfectly. 
(I use the PSVR(iVRy driver) + Nolo controller. )
